dist/tools/vagrant/freebsd/Vagrantfile

```diff
@@ -0,0 +1,38 @@
+$init_riot = <<-INIT_RIOT
+  # vim for xxd
+  pkg install -y bash git gmake gcc cmake afl afl++ \
+      python3 py37-pip py37-scipy py37-pycrypto py37-cython py37-scapy \
+      vim
+  chsh -s /usr/local/bin/bash vagrant
+  if ! [ -d /home/vagrant/RIOT ]; then
+      git clone --recurse-submodules https://github.com/RIOT-OS/RIOT /home/vagrant/RIOT
+  fi
+  chown -R vagrant: /home/vagrant/RIOT
+  curl https://raw.githubusercontent.com/RIOT-OS/riotdocker/master/requirements.txt \
+      > /tmp/requirements.txt
+  su - vagrant -c "pip install --user -r /tmp/requirements.txt"
+  # install most current pexpect to prevent async bug
+  su - vagrant -c "pip install --upgrade --user pexpect"
+  grep -q "export MAKE=gmake" /home/vagrant/.profile || \
```

Comment:
The build system sanity check does not like these exports, this file needs to be whitelisted somehow